crates/cxx-qt-gen/src/writer/cpp/header.rs

```diff
@@ -86,17 +86,11 @@ fn qobjects_header(generated: &GeneratedCppBlocks) -> Vec<String> {
             static_assert(::std::is_base_of<QObject, {ident}>::value, "{ident} must inherit from QObject");
             {namespace_end}
 
-            namespace {namespace_internals} {{
```

Comment:
probably add an entry in the `CHANGELOG.md` saying it was removed, incase anyone was using it.

Reply:
Done